### DISCUSSION TITLE
cfw iplist的疑问

### DISCUSSION THREAD

**Discussion by k79e** [SeaHOH/GotoX#158] [问答/咨询] [answered]
这个上面写的是.0结尾的 会自动变网段.

那我要是填个不是.0结尾的 是会自动变网段 还是就单个ip呢?

还有就是网段内的ip 他可能不是全在线的 自动的那个他有探测功能么? 如果接上了个不在线的ip那岂不是会卡一下?

我怎么能输入指定的ip列表?(都是单个的)

要指定ip还是老样子就用[iplist]那样的就行了是吧.

**Comment by SeaHOH** [comment, marked as answer]
> 那我要是填个不是.0结尾的 是会自动变网段 还是就单个ip呢?

都会扩展 256 个。

> 还有就是网段内的ip 他可能不是全在线的 自动的那个他有探测功能么? 如果接上了个不在线的ip那岂不是会卡一下?

连接 IP 时会进行判断，首次连接的确可能卡，但通常不会太明显。

> 我怎么能输入指定的ip列表?(都是单个的)

当前为了尽量增加可用 IP 数量，没有不扩展 IP 数量的选项。
要禁用数量扩展功能，~直接注释 CFWFetch.py 文件中包含 `explode_ip` 的行即可~。(此处有误，已添加相关选项 a1cb4b1565efe5b3c9de7ed5d8edc965f3df0a89)

> 要指定ip还是老样子就用[iplist]那样的就行了是吧.

连接 CFW 代理的列表是独立专用的，无法使用自定义功能替换。

**Comment by k79e** [comment]
> 连接 CFW 代理的列表是独立专用的，无法使用自定义功能替换。

那我没法给cfw用sni了么
那要是在actionfilter里面 给cf的域名让他走iplist 这2个功能会不会冲突呢!?

**Comment by SeaHOH** [comment]
> > 连接 CFW 代理的列表是独立专用的，无法使用自定义功能替换。
> 
> 那我没法给cfw用sni了么
> 那要是在actionfilter里面 给cf的域名让他走iplist 这2个功能会不会冲突呢!?

这条说的是 [iplist] 设置，CFW 本身那个 iplist 可以随便设置。
当然你想要使用 SNI 代理，需要先手动修改停用数量扩展功能。
不过我觉得没必要，要匿名也不用它啊。

再多解释一句，其它地方设置后只影响直接访问，CFW 代理不受影响。

**Comment by k79e** [comment]
所以actionfilter里面就算让cfw域名走指定ip也无效是吧. 

好了我应该明白了 果然是无效的
